readme.md

```
@@ -49,10 +49,35 @@ npm run dev
+  "_id": "settings",
+  "showUserPasswordLogin": false,
+  "showIdpLogin": true,
+  "idpName": "Local IdP KeyCloak (this appears on the login button)",
+  "idpAppId": "Fauxton",
+  "idpUrl": "http://localhost:8090/realms/empire"
+}
+```
+
+and
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)